cluster/tenant-migrations/000001_tenant-initial.up.sql

```diff
@@ -104,7 +104,7 @@ CREATE TABLE permissions_resources
             REFERENCES permissions
             ON DELETE CASCADE,
     bucket_name      VARCHAR(64)                            NOT NULL,
-    path             VARCHAR(512)                           NOT NULL,
+    pattern          VARCHAR(512)                           NOT NULL,
```

Comment:
for the next prs we need new migrations instead of changing the first one.

Reply:
yeah, we need the `migrate` or `upgrade` command for that